pkgs/development/haskell-modules/configuration-nix.nix

```diff
@@ -105,6 +105,12 @@ self: super: builtins.intersectAttrs super {
     configureFlags =  "--extra-include-dirs=${pkgs.rdkafka}/include/librdkafka";
   });
 
+  # library has hard coded directories that need to be removed https://github.com/NixOS/nixpkgs/pull/29109
```

Comment:
Mentioning the URL of this pull request here is not necessary because the git history will contain that information when the PR is merged. What would be nice, though, is to report that issue *upstream* and to add an URL to that upsteam bug report in that comment instead.